### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by fedepaol**
Bumping a few dependencies and the go version to benefit from fixes.

### CODE REVIEW

File: frr-tools/metrics/exporter.go [hunk 1]
```diff
@@ -24,9 +23,8 @@ import (
 )
```
Comment (oribon) [inline frr-tools/metrics/exporter.go:24]: nit: metrics binary was added by mistake?

Comment (fedepaol) [reply frr-tools/metrics/exporter.go:24]: yep, my bad

Comment (fedepaol) [reply frr-tools/metrics/exporter.go:24]: removed

**Review by oribon — APPROVED** [review]
lgtm

**Pull request merged**